pkg/transforms/vapbinding.go

```diff
@@ -80,6 +80,8 @@ func VapBindingResourceBuilder(v *unstructured.Unstructured) *VapBindingResource
 		return binding
 	}
 
+	// A property must be updated, otherwise the reconcile does not update the edges.
+	binding.node.Properties["_paramRef"] = string(paramRefBytes)
```

Comment:
Can you change where we store this data? The `node.Metadata` map is used to store any data that we need later to compute edges, but it won't get synched with the hub. Everything in `node.Properties` will get added to the search database.
```suggestion
	binding.node.Metadata["_paramRef"] = string(paramRefBytes)
```

Reply:
I could put it in Metadata, but on its own there it doesn't update the edges. I think I could update https://github.com/stolostron/search-collector/blob/eeab67249590613b4a9399abcd53dc17aef290be/pkg/reconciler/reconciler.go#L337 to also check the metadata, or add this resource type as an exception...

Thanks to the test encouraged by @yiraeChristineKim this is easy to check :laughing:

Reply:
Yes, let's update the reconciler code to check the metadata for this change.
I can't recall the exact reason why we don't check changes in metadata, so let's keep the new case as narrow as possible.
```
ne.Node.Properties["kind"] == "VAPBinding" && ne.Node.Metadata["paramRef"] != previousNode.Metadata["paramRef"]
```

Reply:
Updated